services/QuillLMS/app/services/stripe_integration/webhooks/checkout_session_expired_event_handler.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+module StripeIntegration
+  module Webhooks
+    class CheckoutSessionExpiredEventHandler < EventHandler
+      def run
+        stripe_checkout_session&.update!(expiration: DateTime.now.utc)
```

Comment:
Same note as my previous about deleting this record instead of updating expiration.